CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Release History
 
+## 1.28.0 (TBD)
+
+### Snowpark Python API Updates
+
+#### Removed Features
+
+- Removed support for Snowpark Python function `snowflake_cortex_summarize`. Users can use the Summarize function from Snowflake.Cortex instead.
+- Removed support for Snowpark Python function `snowflake_cortex_sentiment`. Users can use the Sentiment function from Snowflake.Cortex instead.
```

Comment:
Removing public API is BCR change, we should discuss, please do not merge the PR